docs/connectors/slack.md

```diff
@@ -72,17 +74,12 @@ connectors:
 
 You need to choose between two backends. The [Events API](https://api.slack.com/apis/connections/events-api) or [Socket Mode](https://api.slack.com/apis/connections/socket).
 
-If you are unsure which one is the best for you, [Slack Faq](https://api.slack.com/faq#events_api) provide differences between those two.
-
-```eval_rst
-.. note::
-  You should follow the instructions for the Event API first when configuring your slack connector, even if you are planning
-  on using slack in Socket Mode.
-```
+> **_NOTE:_** While it is not mandatory to configure the Event API before using Slack in Socket Mode, we recommend setting up the Event API for its full feature set.
```

Comment:
Can you put this back inside the `eval_rst` block and use the note directive as it was before?

Reply:
yeah of course. Done!

Reply:
Hi @jacobtomlinson! If there something else with this pr to fix, let me know